prac_06/guitar_test.py

```diff
@@ -0,0 +1,15 @@
+"""
+Guitar testing program
+Estimate: 10 minutes
+Actual:   10 minutes
+"""
+
+from guitar import Guitar
+
+guitar = Guitar("Gibson L-5 CES", 1922, 16035)
+another_guitar = Guitar("Another Guitar", 2013, 16035)
+
+print(f"Gibson L-5 CES get_age() - Expected 100. Got {guitar.get_age()}")
+print(f"Another Guitar get_age() - Expected 9. Got {another_guitar.get_age()}")
+print(f"Gibson L-5 CES is_vintage() - Expected True. Got {guitar.is_vintage()}")
+print(f"Another Guitar is_vintage() - Expected False. Got {another_guitar.is_vintage()}")
```

Comment:
if refactored into a function use the if __name__ == "__main__" (or any other function name) so that the code will only run the test when this file is run.